src/ui/simulator/windows/inspector/frame.cpp

```diff
@@ -791,8 +791,7 @@ void Frame::apply(const InspectorData::Ptr& data)
         // Area color
         Accumulator<PAreaColor>::Apply(pPGAreaColor, data->areas);
         // Adequacy patch
-        if (!multiple)
-            Accumulator<PAdequacyPatchMode>::Apply(pPGAreaAdequacyPatchMode, data->areas);
+        Accumulator<PAdequacyPatchMode>::Apply(pPGAreaAdequacyPatchMode, data->areas);
```

Comment:
It doesn't seem to work when I tested it. When I select multiple areas in the UI and change the AdequacyPatchMode, only one of them get affected. And I can't figure out why.

Reply:
solved